sw/device/tests/sim_dv/alert_handler_entropy_test.c

```diff
@@ -32,15 +32,16 @@ static void alert_handler_config(void) {
 
   // Enable all incoming alerts and configure them to classa.
   // This alert should never fire because we do not expect any incoming alerts.
-  for (int i = 0; i < ALERT_HANDLER_PARAM_N_ALERTS; ++i) {
+  for (dif_alert_handler_alert_t i = 0; i < ALERT_HANDLER_PARAM_N_ALERTS; ++i) {
```

Comment:
My reading of the loop is that it's trying to generate every possible `dif_alert_handler_alert_t` value. Enums are just `int`, so I figured we might as well use the type. I admit it looks a little weird to index into `alerts` with the enum value, but this is legal and avoids casts.

Happy to change it to this if you'd prefer. The cast would be necessary to avoid implicit sign conversion.

```c
  for (size_t  i = 0; i < ALERT_HANDLER_PARAM_N_ALERTS; ++i) {
    alerts[i] = (dif_alert_handler_alert_t) i;
    alert_classes[i] = kDifAlertHandlerClassA;
  }
```

Reply:
ah right, sorry I misread this.